python/hail/context.py

```diff
@@ -346,7 +346,41 @@ def write_partitioning(self, path):
     @handle_py4j
     def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npartitions=None,
                    sites_only=False, store_gq=False, pp_as_pl=False, skip_bad_ad=False):
-        """Import .vcf files as VariantDataset
+        """Import .vcf files as :py:class:`.VariantDataset`
+
+        **Examples**
+
+        >>> vds = hc.import_vcf('data/example2.vcf.bgz')
+
+        **Notes**
+
+        Hail is designed to be maximally compatible with files in the `VCF v4.2 spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
+
+        :py:meth:`~hail.HailContext.import_vcf` takes a list of VCF files to load. All files must have the same header and the same set of samples in the same order
+        (e.g., a dataset split by chromosome). Files can be specified as `Hadoop glob patterns <https://hail.is/hail/other_resources.html#hadoop-glob-patterns>`_.
+
+        Ensure that the VCF file is correctly prepared for import: VCFs should either be uncompressed (*.vcf*) or block compressed
+        (*.vcf.bgz*).  If you have a large compressed VCF that ends in *.vcf.gz*, it is likely that the file is actually block-compressed,
+        and you should rename the file to ".vcf.bgz" accordingly. If you actually have a standard gzipped file, it is possible to import
+        it to Hail using the ``force`` optional parameter. However, this is not recommended -- all parsing will have to take place on one node because
+        gzip decompression is not parallelizable. In this case, import could take significantly longer.
+
+        Hail makes certain assumptions about the genotype fields, see `Representation <https://hail.is/hail/representation/hail.representation.Genotype.html>`_. On import, Hail filters
```

Comment:
This links works, but we should try not to have the website path hardcoded. Double check this syntax works locally first:

```
:class:`Representation <hail.representation.Genotype>`
```